app/helpers/mustache_helper.rb

```diff
@@ -394,6 +394,23 @@ def content
     end
   end
 
+  def collection_filter_options
```

Comment:
Assignment Branch Condition size for collection_filter_options is too high. [17.15/15]<br>Method has too many lines. [14/10]